terraform/modules/aws-brig-email-sending/README.md

```diff
@@ -0,0 +1,18 @@
+Terraform module: Brig to enable email sending
+==============================================
+
+This module creates AWS resources (SES, SQS, SNS) enabling Brig to send emails.
```

Comment:
```suggestion
Wire-server's "brig" components needs to send emails. This can either be done by configuring an SMTP server (Option 1), or by using AWS resources (SES, SQS, SNS) (Option 2).

This terraform module enables brig to send emails using option 2.
```